habitat_baselines/rl/ddppo/policy/resnet_policy.py

```diff
@@ -119,15 +119,22 @@ def __init__(
         else:
             self._n_input_depth = 0
 
+        if "semantic_category" in observation_space.spaces:
+            self._n_input_semantic_category = observation_space.spaces["semantic_category"].shape[2]
+            spatial_size = observation_space.spaces["semantic_category"].shape[0] // 2
+            assert self._n_input_semantic_category == 3, "ResNetEncoder only supports RGB values from SemanticCategory sensor!"
```

Comment:
Why `semantic_category` why it has RGB values, not int id of the category?

Reply:
Right now, the semantic category sensor outputs an RGB image by default. For each goal category, there is a unique color. For all other categories and backgrounds, the pixels are set to zeros.